content/en/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -300,7 +322,15 @@ liveness and readiness, you can use the SRV records of the Pods (
 application will be able to discover the Pods' addresses when they transition
 to Running and Ready.
 
-### Writing to Stable Storage
+If your application wants to find any healthy Pod in a StatefulSet,
+and therefore does not need to track each specific Pod,
```

Comment:
Cannot understand what this is saying ...

Reply:
OK, the intent is to describe applications that want to connect to something like a database made up of stateful Pods, but the client wants an endpoint representing the database cluster. The client in this example is not interested in tracking each potential backend endpoint and maintaining its own view about which of those backends is healthy.

The Kubernetes _answer_ is load balancing (using kube-proxy, or using round-robin DNS), to avoid the client-side load balancing that a client would otherwise need to do.